sdk/storage/README.md

```diff
@@ -36,3 +36,4 @@ additional questions or comments.
 [coc]: https://opensource.microsoft.com/codeofconduct/
 [coc_faq]: https://opensource.microsoft.com/codeofconduct/faq/
 [coc_contact]: mailto:opencode@microsoft.com
+ 
```

Comment:
Remove unrelated change

Reply:
i need it to trigger the storage pipeline